### ISSUE TITLE
Cannot Set Multiple Port Mappings

### ISSUE THREAD

**Issue by therevoltingx** [bflad/chef-docker#370]
It seems that one cannot specify multiple port mappings for the same docker container.
A proposed solution is to add a `ports` for the docker container resource.  
This will allow a user to set an array for ports, then `resource_docker_container.rb` can have:

`attribute :ports, kind_of: Array, default: nil` 

and `provider_docker_container.rb` to have:

~~~
def parsed_ports
-        return [] if new_resource.port.nil?
-        return [] if new_resource.port.empty?
-        Array(new_resource.port)
+        return [] if new_resource.ports.nil?
+        return [] if new_resource.ports.empty?
+        Array(new_resource.ports)
end
~~~

**Comment by brandocorp** [comment]
The provider code for this is in PR #360, and then the attribute in the resource was fixed in PR #368, so I think this should be fully working now.

**Comment by therevoltingx** [comment]
Ok thank you very much, looks like now i'll be able to set an array of ports using the `port` attribute.

Thanks again for fixing this!
